$MirOS$

	This will expand to
	| #define ICONV_CONST <tabs> AM_ICONV */
	otherwise.

--- config.hin.orig	Thu Aug  2 23:24:04 2007
+++ config.hin	Fri Aug  3 15:58:53 2007
@@ -176,7 +176,7 @@
 #undef HAVE___ARGZ_COUNT	/* AM_GNU_GETTEXT */
 #undef HAVE___ARGZ_NEXT		/* AM_GNU_GETTEXT */
 #undef HAVE___ARGZ_STRINGIFY	/* AM_GNU_GETTEXT */
-#undef ICONV_CONST		/* AM_ICONV */
+#undef ICONV_CONST
 #undef IGNORE_CTRL_C		/* FIXME: make tests? */
 #undef INCLUDE_PROTOTYPES	/* CF_SOCKS5 */
 #undef INSTALL_ARGS		/* CF_PATH_PROG(install) */
